packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -933,61 +967,65 @@ class _ViewContentState extends State<_ViewContent> {
-                        child: SafeArea(
-                          top: false,
-                          bottom: false,
-                          child: SearchBar(
-                            autoFocus: true,
-                            constraints: headerConstraints ?? (widget.showFullScreenView ? BoxConstraints(minHeight: _SearchViewDefaultsM3.fullScreenBarHeight) : null),
-                            leading: widget.viewLeading ?? defaultLeading,
-                            trailing: widget.viewTrailing ?? defaultTrailing,
-                            hintText: widget.viewHintText,
-                            backgroundColor: const MaterialStatePropertyAll<Color>(Colors.transparent),
-                            overlayColor: const MaterialStatePropertyAll<Color>(Colors.transparent),
-                            elevation: const MaterialStatePropertyAll<double>(0.0),
-                            textStyle: MaterialStatePropertyAll<TextStyle?>(effectiveTextStyle),
-                            hintStyle: MaterialStatePropertyAll<TextStyle?>(effectiveHintStyle),
-                            controller: _controller,
-                            onChanged: (String value) {
-                              widget.viewOnChanged?.call(value);
-                              updateSuggestions();
-                            },
-                            onSubmitted: widget.viewOnSubmitted,
-                            textCapitalization: widget.textCapitalization,
-                            textInputAction: widget.textInputAction,
-                            keyboardType: widget.keyboardType,
+              shape: effectiveShape,
+              color: effectiveBackgroundColor,
+              surfaceTintColor: effectiveSurfaceTint,
+              elevation: effectiveElevation,
+              child: ClipRect(
+                clipBehavior: Clip.antiAlias,
+                child: OverflowBox(
+                  alignment: Alignment.topLeft,
+                  maxWidth: math.min(widget.viewMaxWidth, _screenSize!.width),
+                  minWidth: 0,
+                  child: FadeTransition(
+                    opacity: viewIconsFadeCurve,
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.stretch,
+                      children: <Widget>[
+                        Padding(
+                          padding: EdgeInsets.only(top: widget.topPadding),
+                          child: SafeArea(
+                            top: false,
+                            bottom: false,
+                            child: SearchBar(
+                              autoFocus: true,
+                              constraints: headerConstraints ?? (widget.showFullScreenView ? BoxConstraints(minHeight: _SearchViewDefaultsM3.fullScreenBarHeight) : null),
+                              padding: WidgetStatePropertyAll<EdgeInsetsGeometry?>(effectiveBarPadding),
```

Comment:
Oh wait I spoke maybe a little too soon, was the prior behavior that SearchBar was not provided any padding? It looks like SearchBar.padding then was by default 16.0 horizontally. Is the change to 8 here breaking?

Reply:
Does this not make the default 8?

https://github.com/flutter/flutter/blob/50a190caec0c2e130d16140b7f4d67ed106dd3f3/packages/flutter/lib/src/material/search_anchor.dart#L1590-L1591

Reply:
Yes, what I was pointing out is that is not the current default. Why is the default changing in this PR?

Reply:
I know the default padding is different between the base `SearchBar` and the `SearchBar` in the `SearchView`. Let me try and untangle this.

Reply:
Actually... That code snippet I sent is from the current `main` branch. So the current default is already 8.

Reply:
The value of 16 is passed in here:

https://github.com/flutter/flutter/blob/50a190caec0c2e130d16140b7f4d67ed106dd3f3/packages/flutter/lib/src/material/search_anchor.dart#L1086

I don't think this behavior has changed

Reply:
The code is really confusing with this. It definitely tripped me up for a while when I was working on it.